backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtGraphics.java

```diff
@@ -55,7 +56,8 @@ public String getName () {
 	CanvasElement canvas;
 	WebGLRenderingContext context;
 	GLVersion glVersion;
-	GL20 gl;
+	GL20 gl20;
```

Comment:
Can we also make these two private/protected?

Reply:
Note: also removing an unused string variable directly underneath these two variables